transactions/executionNodeVersionBeacon/scripts/get_current_execution_node_version.cdc

```diff
@@ -0,0 +1,6 @@
+import ExecutionNodeVersionBeacon from 0x02
```

Comment:
I would make sure these have correct placeholder addresses

Reply:
Oops, thanks. Need to watch those copy-pastes from playground